lib/task-data/schemas/install-coreos.json

```diff
@@ -40,10 +33,16 @@
             "$ref": "types-installos.json#/definitions/UserPassword"
         },
         "ignitionScriptUri": {
-            "$ref": "#/definitions/IgnitionScriptUri"
+            "$ref": "types-installos.json#/definitions/IgnitionScriptUri"
         },
         "progressMilestones": {
             "$ref": "types-installos.json#/definitions/ProgressMilestones"
+        },
+        "vaultToken": {
+            "$ref": "types-installos.json#/definitions/VaultToken"
```

Comment:
If these new options are only valid for CoreOS, I'd suggest you putting them in install-coreos.json, because types-installos.json is designed for those options that shared by multiple OS.